assets/template-collection.css

```diff
@@ -55,28 +47,58 @@
 }
 
 .collection-filters {
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  grid-template-rows: repeat(2, auto);
 }
 
-@media screen and (min-width: 750px) {
-  .collection-filters {
-    display: block;
-  }
+.collection-product-count {
+  align-self: center;
+  font-size: 1.4rem;
+  line-height: 1.4;
+  margin: 0;
+  text-align: right;
+}
+
+.collection.loading .collection-product-count {
+  visibility: hidden;
+}
 
+.active-facets-mobile {
+  margin-bottom: 0.5rem;
+}
+
+@media screen and (min-width: 750px) {
   .collection-filters > * + * {
     margin-top: 0;
-    margin-left: 2rem;
+  }
+
+  .collection-product-count {
+    align-self: flex-end;
+  }
+
+  .active-facets-mobile + .collection-product-count {
+    grid-column-start: 2;
+    grid-row-start: 2;
+    text-align: right;
+  }
```

Comment:
Is this needed? It's hidden when we're on desktop

[![alt](https://screenshot.click/04-26-y3ww9-nrv19.png)](https://screenshot.click/04-26-y3ww9-nrv19.png)

Reply:
The `.large-up-hide` is added conditionally in Liquid. Depending on the `collapse_on_larger_devices` setting, the "mobile" product count may also visible on desktop:
```
{% unless section.settings.collapse_on_larger_devices %} medium-hide large-up-hide{% endunless %}
```